lib/app.jsx

```diff
@@ -489,7 +522,10 @@ export const App = connect( mapStateToProps, mapDispatchToProps )( React.createC
 								selectedTag={state.tag}
 								tags={state.tags}
 								onOutsideClick={this.onToolbarOutsideClick} />
-							<div className="source-list theme-color-bg theme-color-fg">
+							<div className="source-list theme-color-bg theme-color-fg" ref={ ( ref ) => {
+								if ( !ref ) return;
+								ref.style.flexBasis = sourceList.width + 'px';
```

Comment:
Let's create a method on the class to set the ref like this…

```js
storeSourceList( ref ) => this.sourceList = ref;

<div className="source-list …" ref={ this.storeSourceList }>…</div>
```

and then the mouse move handler can directly reference `this.sourceList`. if it doesn't exist yet then we can safely ignore the move event because the underlying node won't exist yet either. the handler can safely reference `this.sourceList`

also, wtf is going on with setting the stye `ref.style.flexBasis` _inside of the `ref` function_?